src/plugins/intel_cpu/src/graph.h

```diff
@@ -20,6 +20,9 @@
 #include "openvino/runtime/so_ptr.hpp"
 #include "proxy_mem_mgr.h"
 
+#define THROW_WITH_NODE_ERR(node, msg) OPENVINO_THROW("Error with node: ", node->getName(), " of type ", node->getTypeStr(), ": ", msg)
```

Comment:
Can we instead implement operator << for the node, which basically does the same?
I pretty sure this will be convenient for 95% of the similar use cases (i.e. DEBUG_LOG) 
We do have such operator for debug caps but currently it prints to much information I would say.
So, this operator should print name + type instead and, if one needs more information, we just need to introduce some wrapper method to include more information, like verbose(node)

Reply:
At first, this PR is not about debug. The point is just to have more information in the case of exception raising directly in output independently on mode.
I cannot realize how << operator may be _instead_ of this macros. All that is going on is wrapping of exception raising with similar pattern.

Reply:
Done